crowbar_framework/config/boot.rb

```diff
@@ -55,7 +55,7 @@
   gem "dotenv", version: "~> 1.0"
   require "dotenv"
 
-  gem "hashie", version: "~> 2.1"
+  gem "hashie", version: "~> 3.4"
```

Comment:
yeah, I still think, but this patch is more urgent than the other one. I expect the other one to be discussed for a couple of months..